gh2/csv.py

```diff
@@ -206,25 +214,23 @@ def set_headers(repo, labels=None):
         headers.extend('Label: ' + label.name for label in labels)
     return headers
 
+def get_all_labels(repositories, token):
+    labels = []
+    for repository in repositories:
+        repo_owner, repo_name = repository.split('/', 1)
+        repo = get_repo(repo_owner, repo_name, token)
+        repo_labels = get_repo_labels(repo)
+        for repo_label in repo_labels:
+           if repo_label.name not in [l.name for l in labels]:
```

Comment:
the list of label objects that this function returns (as additional_labels) can be/is reused elsewhere so I didn't want to discard the objects inside the function